admin/src/components/Versions/index.js

```diff
@@ -87,7 +87,8 @@ const Versions = () => {
         return;
       }
 
-      const selectedVersion = data.find((v) => v.versionNumber === value);
+      // fixed bug when version being iterated was not the same type as the version being selected (string != number)
+      const selectedVersion = data.find((v) => v.versionNumber === +value);
```

Comment:
Please, consider using this syntax `Number(value)` as it is more readable in my opinion.

Reply:
Change made in [#53cd954](https://github.com/notum-cz/strapi-plugin-content-versioning/pull/90/commits/53cd9548ada04b52342ed5865be7e83f0b24d774)